docs/Manual/04 Fusing.md

```diff
@@ -12,18 +12,17 @@ import accera as acc
 schedule = acc.fuse(schedule0, schedule1, ...)
 ```
 
-*Full fusing* is the most straightforward form of fusing, where each dimension is fused with the corresponding dimension from the other schedules.
+*Full fusing* is the most straightforward fusing where each dimension is fused with the corresponding dimension from other schedules. 
 
 ### Full fusing of same-shaped iteration spaces
-First, consider the simplest case, where we fuse schedules whose iteration spaces have identical shapes. The fused schedule `schedule` gets a new dimension, called the *fusing dimension*, which did not exist in the original schedules. By default, the fusing dimension is the first dimension in the fused schedule and its size equals the number of schedules that were fused. The first slice along the fusing dimension contains a copy of `schedule0`, the second slice contains a copy `schedule1`, and so on. Since the fusing dimension is the first dimension, the fused schedule is logically equivalent to fully executing `schedule0`, followed by `schedule1`, and so on. To interleave the original schedules, we apply additional transformations to the fused schedule.
+Initially, consider the simplest case where we fuse schedules with identical shapes of iteration spaces. This fusing assigns a new dimension called *fusing dimension* to the fused schedule `schedule` that does not exist in the original schedules. By default, the fusing dimension is the first dimension in the fused schedule, having a size equal to the number of fused schedules. The slices along the fusing dimension contain a copy of `schedule0`, `schedule1`. The first slice along the fusing dimension contains a copy of `schedule0`, the second slice contains a copy of `schedule1`, and so on. Since the fusing dimension is the first dimension, the fused schedule is logically equivalent to fully executing `schedule0`, followed by `schedule1`, and so on. We apply additional transformations to the fused schedule to interleave the original schedules.
```

Comment:
"By default, the fusing dimension is the first dimension in the fused schedule. Its size is equal to the number of fused schedules."